src/modules/authentication/server.ts

```diff
@@ -0,0 +1,13 @@
+import passport from 'passport';
+import { GenerateStrategy, UnsupportedAuthenticationMethodError } from './common';
+
+export default (app, moduleConfig) => {
+  import(`./strategies/${moduleConfig.authMethod}.strategy.ts`)
```

Comment:
I noticed several times, it's common to use dynamic imports depending on some variable. What do you think about this approach? In fact, if a developer wants to add another auth method (for example: OpenID), he need to just create file with convention above: `openid.strategy.ts` with a function of a type `GenerateStrategy`

Reply:
Isn't it adding the whole passport to the client bundle anyway? I belive it's not worth it if we can achieve the same logick anyway (I belive we can as you had this http password ttoken genreration code in a 1liner helper)

Reply:
Is a `server.ts` file used in frontend as well? If so, then you're right and I'll remove these dependencies :)

Reply:
No, it shouldn't. So you're right we're safe here :)